### PR TITLE
[Purrfect-UI] TopBar

### PR DESCRIPTION
**Pull request by Rcaradec**

### CODE REVIEW

**Comment by MatouTechnologies** [comment]
This pull request has been deployed to Vercel.
<table>
  <tr>
    <td><strong>✅ Preview:</strong></td>
    <td><a href='https://matou-technologies-purrfect-ui-storybook-38.vercel.app'>https://matou-technologies-purrfect-ui-storybook-38.vercel.app</a></td>
  </tr>
  <tr>
    <td><strong>🔍 Inspect:</strong></td>
    <td><a href='https://vercel.com/matou-technologies/purrfect-ui/CLBm16eBTyxShecwzdB2Uiqk8Lqj'>https://vercel.com/matou-technologies/purrfect-ui/CLBm16eBTyxShecwzdB2Uiqk8Lqj</a></td>
  </tr>
</table>


<hidden purpose="for-rewritable-pr-comment-action-use" value="deploy-preview-storybook"></hidden>

File: src/display/TopBar/TopBar.tsx [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+  sectionRightWidth?: string;
+};
+
+type TopBarSpacing =
+  | "flex-start"
+  | "flex-end"
+  | "center"
+  | "space-around"
+  | "space-evenly"
+  | "space-between";
```
Comment (MarcoPOtard) [inline src/display/TopBar/TopBar.tsx:77]: I think those 3 didn't change anything
"space-around"
  | "space-evenly"
  | "space-between";

File: src/display/TopBar/TopBar.spec.tsx [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+      />,
+    );
+    expect(screen.getByText("LEFT")).toBeInTheDocument();
+    expect(screen.getByText("CENTER")).toBeInTheDocument();
+    expect(screen.getByText("RIGHT")).toBeInTheDocument();
+  });
+
+  it("should render 1 child", () => {
+    render(<TopBar leftSection={<div>LEFT</div>} />);
+    expect(screen.getByText(/LEFT/)).toBeInTheDocument();
```
Comment (MarcoPOtard) [inline src/display/TopBar/TopBar.spec.tsx:27]: for the "getByText(/LEFT/)"
you have the choice, either you write it  : getByText("LEFT")
or you write it with a final "i" : getByText(/LEFT/i)
But I think in the getByText function with double quote is correct
Check every test that use it

File: src/display/TopBar/TopBar.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+    leftFlex: {
+      control: { type: "number", min: 0, max: 3 },
+    },
+    centerFlex: {
+      control: { type: "number", min: 0, max: 3 },
+    },
+    rightFlex: {
+      control: { type: "number", min: 0, max: 3 },
+    },
+    "aria-label": {
```
Comment (MarcoPOtard) [inline src/display/TopBar/TopBar.stories.tsx:58]: I thnik this one is useless, in args aria-label is a string

File: src/display/TopBar/TopBar.tsx (hunk 1 above)
Comment (Rcaradec) [reply src/display/TopBar/TopBar.tsx:77]: removed

File: src/display/TopBar/TopBar.spec.tsx (hunk 2 above)
Comment (Rcaradec) [reply src/display/TopBar/TopBar.spec.tsx:27]: fixed

File: src/display/TopBar/TopBar.stories.tsx (hunk 3 above)
Comment (Rcaradec) [reply src/display/TopBar/TopBar.stories.tsx:58]: removed

**Review by MarcoPOtard — APPROVED** [review]

**Pull request merged**